pkg/controllers/management/node/cleanup.go

```diff
@@ -411,9 +411,8 @@ func (m *Lifecycle) userNodeRemoveCleanup(obj *v3.Node) (runtime.Object, error)
 		}
 
 		annos[userNodeRemoveCleanupAnnotation] = "true"
-		newObj.SetAnnotations(annos)
```

Comment:
Why is this line dropped? 
It makes the logic clear although we know that Map in Go is passed by reference. 😄

Reply:
Fixed!